### PR TITLE
Feature drag and drop

### PR DESCRIPTION
**Pull request by pochretien**
Change the icon  more
Change the icon drag and drop
Add style for the drag and drop in tableview
add a demo with a functional drag and drop

### CODE REVIEW

**Comment by pochretien** [comment]
screen shot 2017-04-06 at 10 31 11 am

**Comment by pochretien** [comment]
screen shot 2017-04-06 at 10 31 17 am

File: docs/_components/tables/02_table-drag-and-drop.html [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+        e.dataTransfer.setData('text', 'drag');
+    });
+
+    dragAndDropTableRow.on('dragenter', function(e) {
+        dragAndDropTableRow.removeClass('drag-over');
+        $(this).addClass('drag-over');
+        e.preventDefault();
+    });
+
+    dragAndDropTableRow.on('dragend', function(e) {
```
Comment (sroy3) [inline docs/_components/tables/02_table-drag-and-drop.html]: You could probably merge all of you events together and do:
```
dragAndDropTableRow.on('dragleave dragenter dragend dragover dragleave drop', function(e) {
  e.preventDefault();
});
```
Then you can also add the ones with more code after without repeating the `e.preventDefault();`

File: scss/tables/table-drag-and-drop.scss [hunk 2]
```diff
@@ -0,0 +1,43 @@
+table.drag-and-drop-table-view {
+  [draggable] {
```
Comment (MichelML) [inline scss/tables/table-drag-and-drop.scss]: nice! didn't know the draggable attribute existed

File: scss/tables/table-drag-and-drop.scss [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+  .drag-feedback {
+    position: relative;
+  }
+
+  .handle {
+    padding: 0;
+
+    text-align: center;
+
+    cursor: move; // grab is not available for ie 11
```
Comment (MichelML) [inline scss/tables/table-drag-and-drop.scss]: maybe put the reference here https://msdn.microsoft.com/library/aa358795(v=vs.85).aspx gg for finding out

**Pull request merged**